collect_app/src/main/java/org/odk/collect/android/adapters/FileArrayAdapter.java

```diff
@@ -103,4 +105,22 @@ public View getView(int position, View convertView, ViewGroup parent) {
         }
         return v;
     }
+    @Override
+    public boolean isEnabled(int position) {
+        if (areAllItemsEnabled()) {
+            return true;
+        }
+        if(!enabled){
+            return false;
+        }
+        return true;
+    }
+    @Override
+    public boolean areAllItemsEnabled() {
+        return enabled;
+    }
+
+    public void enableAdapterClick(boolean enabled) {
+        this.enabled = enabled;
+    }
 }
```

Comment:
@nribeka What exactly does that no new line at end of file mean? Is it necessary to have a new line?